src/service/transaction.service.ts

```diff
@@ -51,7 +51,11 @@ export class TransactionService {
             } else {
                 transactionFormatted += transaction.recipient.toHex();
             }
-            transactionFormatted += transaction.message.payload.length > 0 ? ' Message:\"' + transaction.message.payload + '\"' : '';
+            try {
```

Comment:
Could you try to fix it without using try/catch? Thanks!

Reply:
ok,